int16 r_side
int16 r_front 
int16 l_front
int16 l_side
int16 sum
int16 sum_forward